science/cdo/Portfile

```diff
@@ -5,7 +5,7 @@ PortGroup                   mpi 1.0
 PortGroup                   legacysupport 1.0
 
 name                        cdo
-version                     2.4.3
+version                     2.4.4
 revision                    0
 platforms                   darwin
```

Comment:
`platforms darwin` is implicit now - we could drop this line as per `lint` output.

Reply:
I removed it in the following commit.
Thanks for the suggestion.